test/sql/grant_hypertable.sql

```diff
@@ -0,0 +1,45 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+\c :TEST_DBNAME :ROLE_SUPERUSER
+
+CREATE TABLE conditions(
+    time TIMESTAMPTZ NOT NULL,
+    device INTEGER,
+    temperature FLOAT
+);
+
+-- Create a hypertable and show that it does not have any privileges
+SELECT * FROM create_hypertable('conditions', 'time', chunk_time_interval => '5 days'::interval);
+INSERT INTO conditions
+SELECT time, (random()*30)::int, random()*80 - 40
+FROM generate_series('2018-12-01 00:00'::timestamp, '2018-12-10 00:00'::timestamp, '1h') AS time;
+\z conditions
+\z _timescaledb_internal.*chunk
+
+-- Add privileges and show that they propagate to the chunks
+GRANT SELECT, INSERT ON conditions TO PUBLIC;
+\z conditions
+\z _timescaledb_internal.*chunk
+
+-- Create some more chunks and show that they also get the privileges.
+INSERT INTO conditions
+SELECT time, (random()*30)::int, random()*80 - 40
+FROM generate_series('2018-12-10 00:00'::timestamp, '2018-12-20 00:00'::timestamp, '1h') AS time;
+\z conditions
+\z _timescaledb_internal.*chunk
+
+-- Revoke one of the privileges and show that it propagate to the
+-- chunks.
+REVOKE INSERT ON conditions FROM PUBLIC;
+\z conditions
+\z _timescaledb_internal.*chunk
+
+-- Add some more chunks and show that it inherits the grants from the
```

Comment:
also need a test that shows that a user (non super user) can SELECT but not insert.  The tests have additional roles DEFAULT_PERM_USER that can be used for this .

Reply:
It seems redundant. The current list shows that permissions are propagated correctly and adding select tests would just check that the permissions support actually works. I think these tests are about testing that permissions are propagated, not testing that permissions are correctly implemented in PostgreSQL.